dbfit-java/informix/src/test/java/dbfit/environment/InformixBigDecimalTransformerTest.java

```diff
@@ -0,0 +1,42 @@
+package dbfit.environment;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.lang.Double;
+import java.sql.SQLException;
+
+import dbfit.util.NormalisedBigDecimal;
+
+public class InformixBigDecimalTransformerTest {
+
+    private final InformixBigDecimalTransformer bdt = new InformixBigDecimalTransformer();
+    private final BigDecimal inBigDec = new BigDecimal("123.45");
+
+    @Test
+    public void transformsBigDecimaltoBigDecimalTest() throws SQLException {
+        BigDecimal bd = new BigDecimal(inBigDec.toString());
+        BigDecimal outBigDec = (BigDecimal) bdt.transform(bd);
+        assertEquals(outBigDec, inBigDec);
+    }
+
+    @Test
+    public void transformsNormalisedBigDecimaltoBigDecimalTest() throws SQLException {
+        NormalisedBigDecimal nbd = new NormalisedBigDecimal(inBigDec);
+        BigDecimal outBigDec = (BigDecimal) bdt.transform(nbd);
+        assertEquals(outBigDec, inBigDec);
+    }
+
+    @Test
+    public void rejectsNonNormalisedBigDecimalInputTest() {
```

Comment:
I think I've mentioned about JUnit expected exceptions capability. May we try to see if it can simplify this part. Also I'm not sure about having that as `SQLException` - maybe we better stick with the original approach (UnsupportedOperationException).